Makefile

```diff
@@ -88,6 +82,9 @@ pypi: wheel source-pypi develop
 	@echo " twine upload -u <PYPI_USERNAME> PYPI_UPLOAD/*.{tar.gz,whl}"
 	@echo
 
+python_build: pip
```

Comment:
Although the `all` section is not very up to date, would be nice to have `python_build` target described in the `all` section as well.